packages/thirdweb/src/wallets/in-app/native/helpers/storage/local.ts

```diff
@@ -112,6 +112,22 @@ export async function setDeviceShare({
   return deviceShare;
 }
 
+export async function removeDeviceShare({
+  clientId,
+}: {
+  clientId: string;
+}): Promise<void> {
+  const userDetails = await getWalletUserDetails(clientId);
+
+  if (!userDetails) {
+    throw new Error("Missing wallet user ID");
```

Comment:
prob dont need to throw here? maybe just return early

Reply:
We won't be able to remove the device share though (even though one could exist)